### PR TITLE
Person addon

### PR DESCRIPTION
**Pull request by zainab-786**
Added postal code validations.

### CODE REVIEW

File: person/views/custom_client_view.xml [hunk 1]
```diff
@@ -13,14 +13,12 @@
...
+                        <field name="postalcode"  required="1"/>
                     </group>
                 </sheet>
             </form>
         </field>
     </record>
-
-
     <record id="custom_client_tree_view" model="ir.ui.view">
         <field name="name">person.person.tree</field>
```
Comment (MORPSmcaf) [inline person/views/custom_client_view.xml]: <field name="name">custom.client.tree</field>

File: person/views/custom_client_view.xml [hunk 2]
```diff
@@ -32,24 +30,15 @@
...
-    <menuitem id="menu_root_person"
-              name="Custom Client"/>
-
-    <menuitem id="menu_person_id1"
-              name="Client Details"
-              parent="menu_root_person"
-              action="action_client_example"/>
-
-
+    <menuitem id="menu_root_person" name="Custom Client"/>
```
Comment (MORPSmcaf) [inline person/views/custom_client_view.xml]: it was ok previously in column not row

 <menuitem id="menu_root_person"
              name="Custom Client"/>

    <menuitem id="menu_person_id1"
              name="Client Details"
              parent="menu_root_person"
              action="action_client_example"/>

File: person/views/custom_client_view.xml [hunk 3]
```diff
@@ -32,24 +30,15 @@
                 <field name="email" required="1"/>
                 <field name="photo" required="1"/>
                 <field name="gender" required="1"/>
+                <field name="postalcode"  required="1"/>
             </tree>
         </field>
     </record>
```
Comment (MORPSmcaf) [inline person/views/custom_client_view.xml:39]: add one space down

File: person/models/client.py [hunk 4]
```diff
@@ -1,9 +1,16 @@
...
     _inherit = 'abstract.person'
 
-    photo = fields.Char(string="photo")
+    photo = fields.Binary(string="Photo")
+    postalcode = fields.Char(string="Postal Code")
+
+    @api.constrains('postalcode')
+    def _check_postalcode(self):
+        for record in self:
+            if record.postalcode and len(record.postalcode) != 6:
```
Comment (MORPSmcaf) [inline person/models/client.py]: add validation like:
Postal code: enter a valid postal code in format A1A 1A1
not only characters long

**Comment by zainab-786** [comment]
Added validations in the form A1A A1A.

**Comment by zainab-786** [comment]
Added validations in the form A1A A1A.

**Pull request closed without merging**